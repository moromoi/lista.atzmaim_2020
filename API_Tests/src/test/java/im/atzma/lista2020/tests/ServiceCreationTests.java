package im.atzma.lista2020.tests;

import org.approvaltests.Approvals;
import org.testng.annotations.Test;

public class ServiceCreationTests extends TestBase {

    @Test(priority = 1)
    public void creatNewService() {
        int id = app.serviceCreationHelper().createCategory();
        app.serviceCreationHelper().createService(id);

        Approvals.verify(app.serviceCreationHelper().getServiceList());

    }


}
