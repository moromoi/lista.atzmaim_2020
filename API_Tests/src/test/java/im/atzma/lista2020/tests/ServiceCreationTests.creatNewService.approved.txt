Server=nginx
#####
Content-Type=application/json; charset=utf-8
Transfer-Encoding=chunked
Connection=keep-alive
Vary=Accept-Encoding
Set-#####
Set-#####
Strict-Transport-Security=max-age=604800
X-Content-Type-Options=nosniff
X-Robots-Tag=noindex
Content-Security-Policy=upgrade-insecure-requests
X-Powered-By=php/4.3.10-16
Content-Encoding=gzip

[
    {
        "#####
        "name": "ייעוץ",
        "price": "0",
        "duration": 60,
        "color": "#efa5fe",
        "category": {
            "name": "Alternative Therapy Specialists",
            "#####
        }
    },
    {
        "#####
        "name": "פגישת אבחון ראשונה",
        "price": "0",
        "duration": 90,
        "color": "#efa5fe",
        "category": {
            "name": "Alternative Therapy Specialists",
            "#####
        }
    },
    {
        "#####
        "name": "טיפול - 30 דק'",
        "price": "0",
        "duration": 30,
        "color": "#efa5fe",
        "category": {
            "name": "Alternative Therapy Specialists",
            "#####
        }
    },
    {
        "#####
        "name": "טיפול - 45 דק'",
        "price": "0",
        "duration": 45,
        "color": "#efa5fe",
        "category": {
            "name": "Alternative Therapy Specialists",
            "#####
        }
    },
    {
        "#####
        "name": "טיפול - 60 דק', ",
        "price": "0",
        "duration": 60,
        "color": "#efa5fe",
        "category": {
            "name": "Alternative Therapy Specialists",
            "#####
        }
    },
    {
        "#####
        "name": "טיפול - 90 דק'",
        "price": "0",
        "duration": 90,
        "color": "#efa5fe",
        "category": {
            "name": "Alternative Therapy Specialists",
            "#####
        }
    },
    {
        "#####
        "name": "ביקורת",
        "price": "0",
        "duration": 60,
        "color": "#efa5fe",
        "category": {
            "name": "Alternative Therapy Specialists",
            "#####
        }
    },
    {
        "#####
        "name": "שיחה טלפונית",
        "price": "0",
        "duration": 30,
        "color": "#efa5fe",
        "category": {
            "name": "Alternative Therapy Specialists",
            "#####
        }
    },
    {
        "#####
        "name": "TestService",
        "price": "50",
        "duration": 60,
        "color": "#50e3c1",
        "category": {
            "name": "TestCategory",
            "#####
        }
    }
]